### DISCUSSION TITLE
How can I fixed sidebar?

### DISCUSSION THREAD

**Discussion by emirhanbayrak** [tailwindlabs/tailwindcss#10256] [Help]
Hey everyone,
I want to make a simple resume website with tailwindcss. I used grid system for locating the sidebar and content.
As shortly, I used 9 column, and for container 7 column (2 for sidebar/ 7 for content) 

main div

`<body class="grid grid-cols-9 bg-slate-300">`

sidebar div

`<div class="col-span-3 flex h-screen  items-center justify-center bg-emerald-600 lg:col-span-2 lg:col-start-2">`

If I get the point, I want to fix the sidebar same location at the `lg:col-span-2 lg:col-start-2`. Nevertheless, when I choose "fixed" class, sidebar moving "left:0" location. Im gonna show by screenshots

mail view

[https://i.imgur.com/1pFDwnP.png](url)

other view of small devices

[https://i.imgur.com/ZLKAmkh.png](url)

grid view for main view

[https://i.imgur.com/w94HqTE.png](url)

when I chose fixed class

[https://i.imgur.com/tlo1hUY.png](url)

How can I fix this issue with which class or extra codes? Thanks in advance for everyone.

**Comment by wongjn** [comment]
The `fixed` class applies `position: fixed`, which takes the element out of the layout flow and thus out of the grid system. You can try using a `left-*`, `right-*`, `translate-x-*`, `ml-*` or `mr-*` class to position it how you'd like.

**Reply by emirhanbayrak** [reply]
I got it but How can I fix it with grid system? We don't have any solution way?

**Reply by wongjn** [reply]
No, the `fixed` class applies `position: fixed`, which takes the element out of the layout flow and thus out of the grid system. You'd need to use a different `position` value – `static`, `relative` or `sticky` – to position the element with grid.

**Reply by emirhanbayrak** [reply]
thanks so much mate.
